yarn.lock

```diff
@@ -906,7 +906,7 @@ camelcase@^4.1.0:
 
 camelize@^1.0.0:
   version "1.0.0"
-  resolved "https://npm.kopaxgroup.com/camelize/-/camelize-1.0.0.tgz#164a5483e630fa4321e5af07020e531831b2609b"
+  resolved "https://registry.yarnpkg.com/camelize/-/camelize-1.0.0.tgz#164a5483e630fa4321e5af07020e531831b2609b"
```

Comment:
@jacobp100 This change fixes the Travis build errors, and I changed some Travis settings at the same time. Please review!